apps/picsa-apps/dashboard/src/app/modules/translations/pages/home/translations.page.html

```diff
@@ -0,0 +1,61 @@
+<div class="page-content">
+  <div style="display: flex; align-items: center">
+    <h2 style="flex: 1">Translations</h2>
+    <button mat-raised-button color="primary" routerLink="/translations/new">Add New Translation</button>
+  </div>
+  @if(service.translations){
+  <table mat-table [dataSource]="service.translations">
```

Comment:
(+1) Nice to see the mat-table implementation. I'm currently creating a shared component to make it easier to use mat tables as always feels like a lot of boilerplate code to have to write. But think this works well for now so I'll just let you know when the shared component is ready.